source/drivers/driver-compatibility-reference.txt

```diff
@@ -1121,56 +1269,100 @@ Language Compatibility
      - Python 2.5
      - Python 2.6
      - Python 2.7
-     - Python 3.1
-     - Python 3.2
      - Python 3.3
      - Python 3.4
+     - Python 3.5
+     - Python 3.6
```

Comment:
Cool. I'll remove it - I stole the language compatibility for Motor from the http://motor.readthedocs.io/en/stable/requirements.html so y'all may want to update that as well 😺.